blueprints/module-unification-app/files/bower.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We don't emit `bower.json` any longer

Reply:
Hmm, maybe we still need one to get ember-canary build? 🤔

Reply:
LMK if it's possible to opt in to canary in `package.json`. Otherwise, I'll leave this as is for now

Reply:
We can (via `github:emberjs/ember.js`) but it requires npm@5 or yarn@1 I believe.  Seems good to test if you have time?

Reply:
I couldn't seem to get this to work. I'll add to this my list and come back to it in a future PR